Project.toml

```diff
@@ -12,9 +12,9 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-Distances = "≥ 0.8.0"
-NearestNeighbors = "≥ 0.0.3"
-StatsBase = "≥ 0.9.0"
+Distances = "0.8 1"
```

Comment:
This syntax isn't actually supported. I'm not sure it's possible to include all versions between 0.8 and 1 in the current system. Anyway it would be better to include 1.x, since it's unlikely that the move to 1.0 will be more breaking than the move from one minor version to the other (quite the opposite).

Reply:
reverted thanks

Reply:
So we're back to the original issue... :-/